### PR TITLE
EES-2717 more methodology page unit tests

### PR DESCRIPTION
**Pull request by amyb-hiveit**
Adds some more tests for the frontend methodology page.

### CODE REVIEW

File: src/explore-education-statistics-frontend/src/modules/methodologies/__tests__/MethodologyPage.test.tsx [hunk 1]
```diff
@@ -21,57 +21,155 @@ describe('MethodologyPage', () => {
...
-      <MethodologyPage
-        data={{
-          ...testMethodology,
-          notes: [],
-        }}
-      />,
+  test('renders the methodology content', () => {
+    render(<MethodologyPage data={testMethodology} />);
+    const contentAccordion = screen.getAllByTestId('accordion')[0];
+    const contentAccordionSections = within(contentAccordion).queryAllByTestId(
```
Comment (ntsim) [inline src/explore-education-statistics-frontend/src/modules/methodologies/__tests__/MethodologyPage.test.tsx:37]: Could use `getAllByTestId`?

File: src/explore-education-statistics-frontend/src/modules/methodologies/__tests__/MethodologyPage.test.tsx [hunk 2]
```diff
@@ -21,57 +21,155 @@ describe('MethodologyPage', () => {
...
   });
 
-  test(`renders 'Last updated' with the date of the most recent note`, () => {
+  test('renders the methodology annexes', () => {
     render(<MethodologyPage data={testMethodology} />);
-
-    expect(screen.getByTestId('Last updated-value')).toHaveTextContent(
-      '15 September 2021',
+    const annexAccordion = screen.getAllByTestId('accordion')[1];
+    const annexAccordionSections = within(annexAccordion).queryAllByTestId(
```
Comment (ntsim) [inline src/explore-education-statistics-frontend/src/modules/methodologies/__tests__/MethodologyPage.test.tsx:71]: Could use `getAllByTestId`?

File: src/explore-education-statistics-frontend/src/modules/methodologies/__tests__/__data__/testMethodologyData.ts [hunk 3]
```diff
@@ -1,25 +1,92 @@
 import { Methodology } from '@common/services/methodologyService';
 
 export const testMethodology: Methodology = {
-  id: '0d31edaa-e750-4f39-90c4-9e21774f1ef3',
+  id: 'methodology-1',
   title: 'Pupil absence statistics: methodology',
   published: '2021-02-16T15:32:01.2216362',
```
Comment (ntsim) [inline src/explore-education-statistics-frontend/src/modules/methodologies/__tests__/__data__/testMethodologyData.ts:6]: No need to include sub-second time precision

**Review by benoutram — APPROVED** [review]
Checked comments left by @ntsim have been resolved and looks good to me 🙂

**Pull request merged**